package.json

```diff
@@ -4,7 +4,9 @@
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "test": "lerna run test",
-    "build": "lerna run build"
+    "build": "lerna run build",
+    "copy:files": "npx cpx README.md packages/core",
```

Comment:
@SachinHatikankar100 What do you think, maybe we should add `cpx` as a devDependency in the `package.json` instead of using npx?

Reply:
As discussed,  ```cpx``` to be added in devDependency in the ```package.json``` file. Hence, made the necessary changes in this PR.